app/assets/stylesheets/star_show_styling.scss

```diff
@@ -1,15 +1,19 @@
-.accordion-navigation {
-  >a:after {
-    content: "↡";
-    padding-left: 5px;
-    float: right;
-    font-size: 20px;
-  }
-  &.active > a:after {
-    content: "↟";
-  }
+.accordion {
+  color: #fefefe;
```

Comment:
Color literals like `#fefefe` should only be used in variable declarations; they should be referred to via variable everywhere else.